sdkproject/Assets/Mapbox/Unity/MeshGeneration/Modifiers/MeshModifiers/LineMeshModifier.cs

```diff
@@ -12,14 +25,66 @@ namespace Mapbox.Unity.MeshGeneration.Modifiers
 	[CreateAssetMenu(menuName = "Mapbox/Modifiers/Line Mesh Modifier")]
 	public class LineMeshModifier : MeshModifier
 	{
-		[SerializeField]
-		public float Width = 3.0f;
+		#region Constants
```

Comment:
Create a datacontainer for LineModifier Properties (deriving from ModifierProperties) and use SetProperties pattern like other modifiers.

Reply:
etracted only the width for now. we can move jointypes after I finish those and we decided how to do it etc.